tests/integration/devtools/test_callback_validation.py

```diff
@@ -677,7 +677,7 @@ def c2(children):
 
     @app.callback([Output("a", "children")], [Input("c", "children")])
     def c3(children):
-        return children
+        return (children,)
```

Comment:
I assume that this is due to timing changes: The `not a tuple` error started appearing first instead of second and broke the test. Fixing it.